### ISSUE TITLE
string before "CR" is not displayed when reading RSS

### ISSUE THREAD

**Issue by berli0z** [tracking-exposed/facebook#114]
we need to fix these lines https://github.com/tracking-exposed/facebook/blob/4d9fcc233e3a0533ea7d322eb678dbc8fe2fb929/lib/rss.js#L14
https://github.com/tracking-exposed/facebook/blob/4d9fcc233e3a0533ea7d322eb678dbc8fe2fb929/lib/rss.js#L15
https://github.com/tracking-exposed/facebook/blob/4d9fcc233e3a0533ea7d322eb678dbc8fe2fb929/lib/rss.js#L16

**Comment by vecna** [comment]
Yes I agree, but I don't know how we can do a carriage return in RSS. Do you know how? Can you try to change the code in testing server and check if works?

**Comment by berli0z** [comment]
I think we should put together placeholder and description and wrap them inside CDATA: https://stackoverflow.com/questions/1351107/how-to-add-line-breaks-in-rss-feeds
